examples/avatarMocap.tsx

```diff
@@ -39,19 +44,23 @@ type AvailablePoses =
   | 'mocapTurn45'
   | 'mocapTurn90'
 
-const mocapTestData = {} as Record<AvailablePoses, MotionCaptureResults>[]
-getMocapTestData().then((data) => {
-  Object.assign(mocapTestData, data)
-  console.log({ mocapTestData })
-})
-
-const ActivePoseState = createState<AvailablePoses>('mocapTPose')
-
-const MocapAvatar = (props: { userID: UserID }) => {
-  const { userID } = props
-  const entity = useHookstate(UUIDComponent.entitiesByUUIDState[userID + '_avatar']).value
+const poses = [
```

Comment:
now that the mocap test files have moved to this projec we need to update the import glob